config/config_test.go

```diff
@@ -75,7 +78,9 @@ func (s *ConfigSuite) TestRead(c *gc.C) {
 				mustParseKey("lsvcDkapKoFxIyjX9/eQgb3s41KVwPMISFwAJdVCZ70="),
 			},
 		},
-		StatsCacheMaxAge: 3600,
+		StatsCacheMaxAge: config.DurationString{time.Hour},
+		RequestTimeout:   config.DurationString{500 * time.Millisecond},
```

Comment:
Based on the observations of the behaviour under stress tests, anyone objects if we set this to 2s?